utilities/decrypt_data/export_postgres_to_excel_chunks.py

```
@@ -0,0 +1,585 @@
+            base_query = base_query_sql.as_string(self.connection)
+            
+            total_exported_rows = 0
+            writer = None
+            
+            try:
+                for chunk_num in range(total_chunks):
+                    offset = chunk_num * chunk_size
+
+                    # Add LIMIT and OFFSET to query
+                    chunked_query = f"{base_query} LIMIT {chunk_size} OFFSET {offset}"
+                    logger.info(f"Chunk {chunk_num + 1}/{total_chunks}: Fetching rows {offset:,} to {offset + chunk_size - 1:,}")
```

Ruff (0.12.2)

Logging statement uses f-string

(G004)